### PR TITLE
#5 Created Events Section 

### PR DESCRIPTION
**Pull request by Kraniket901**
Added file5.png in src/assets/images
Created section 3 folder containing Section3.js and Section3.css
Added Section3 Component in Homepage.js
Added href=#events in Navbar Component

### CODE REVIEW

**Review by sahil9001 — CHANGES_REQUESTED** [review]
Few points to be noted:
- Add more events so we can see the scrolling functionality of the event bar.
- Card should not have a drop shadow on hover, instead it should be elevated towards user.
- Please check the card font, it is not matching, even the margin and card spacing is improper. The developed website should not differ much from the figma design.

**Comment by sahil9001** [comment]
Only HCs have the right to review the tasks, hierarchy should be maintained. You don't have to add @virajchandra51 for review

**Comment by Kraniket901** [comment]
- Functionality Added
- Changed Card Hover Effect
- Changed Margin and Padding

**Comment by sahil9001** [comment]
Capture
Fix this height, either make it truncated for longer texts like "Hello this is an event...." or else extend height to match the levels.

**Comment by Kraniket901** [comment]
- Fixed Line Height

File: src/components/eventCard/EventCard.css [hunk 1]
```diff
@@ -30,6 +30,9 @@
 .codeutsava__section3-cardtitle{
     font-weight:bolder;
     margin: 0.5rem 0;
+    min-height: 56px;
```
Comment (sahil9001) [inline src/components/eventCard/EventCard.css:33]: Use `3rem` here pixels aren't relative to different sizes

**Review by sahil9001 — CHANGES_REQUESTED** [review]

File: src/components/eventCard/EventCard.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+.codeutsava__section3-card{
+    /* background-color: blueviolet; */
+    padding-bottom: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif; 
+}
+
+.codeutsava__section3-cardtitle{
+    font-weight:bolder;
```
Comment (sahil9001) [inline src/components/eventCard/EventCard.css:31]: Set font-weight to 600 , please look at figma design. You can click on the text in design then go to the inspect tab on top right and use the css provided for reference

File: src/components/eventCard/EventCard.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    margin: 0.5rem 0;
+    min-height: 56px;
+    display: flex;
+    align-items: center;
+}
```
Comment (sahil9001) [inline src/components/eventCard/EventCard.css:36]: Add `line-height` as 24px

File: src/components/eventCard/EventCard.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+.codeutsava__section3-events-container{
+    display: flex;
+    flex-direction: row;
+    width: max-content;
+    overflow-x: scroll;
+    -ms-overflow-style: none;
+    scrollbar-width: none; 
+    gap: 3rem;
```
Comment (sahil9001) [inline src/components/eventCard/EventCard.css:16]: Set gap to `2rem`

**Review by sahil9001 — CHANGES_REQUESTED** [review]

**Comment by Kraniket901** [comment]
Done Sir

**Comment by sahil9001** [comment]
Great work, @Kraniket901 !

**Review by sahil9001 — APPROVED** [review]

**Pull request merged**